src/NewChain/OriginChainInteract.ts

```diff
@@ -125,7 +125,7 @@ export default class OriginChainInteract {
 
     this.logInfo('staking');
     const stakeReceipt = await this.ostGateway.stake(
-      this.initConfig.originStakeAmount,
+      this.initConfig.originStakeAmount.toString(),
```

Comment:
```suggestion
      this.initConfig.originStakeAmount.toString(10),
```

Reply:
`originStakeAmount ` is a string, .toString(10) wouldn't work. We should ideally remove .toString()